app/instance-initializers/raven-setup.js

```diff
@@ -5,6 +5,16 @@ import config from '../config/environment';
 // compatibility.
 const assign = Ember.assign || Ember.merge;
 
+const parseRegexErrors = function parseRegexErrors(ravenOptions) {
+  return Ember.get(ravenOptions, 'ignoreErrors').map((error) => {
+    if (error.startsWith('/') && error.endsWith('/')) {
```

Comment:
This wont match anything but the simplest regexes, for example regexes can have multiple modifiers at the end.